ts/src/index.ts

```diff
@@ -122,6 +123,35 @@ export async function start(config: Config = {}): Promise<void> {
   profiler.start();
 }
 
+
+/**
+ * For debugging purposes. Collects profiles and writes profiles
+ * out to console in the format used to upload the profiles to the API.
+ */
+export async function startLocal(config: Config = {}): Promise<void> {
+  const normalizedConfig = await initConfig(config);
+  profiler = new Profiler(normalizedConfig);
+  
+  while (true) {
+    if(!config.disableHeap) {
+      const heap = await profiler.profile({
+        name: 'HEAP-Profile' + new Date(),
+        profileType: 'HEAP'
+      });
+      console.log(heap);
```

Comment:
Will it print binary content to the console? That's crude if so. I'd either discard the collected profiles or save to temporary files.

Reply:
I'm going to discard the collected profiles.